smtp/send_mail_resource.go

```diff
@@ -164,6 +164,7 @@ func (r *sendMailResource) Create(ctx context.Context, req resource.CreateReques
 	receivers = uniqueAttrValue(receivers)
 	msg := []byte("To: " + strings.Join(asStringList(plan.To.Elements()), ", ") + "\r\n" +
 		"Cc: " + strings.Join(asStringList(plan.Cc.Elements()), ", ") + "\r\n" +
+		"From: " + from + "\r\n" +
```

Comment:
line no 174 already takes care of added the `From` information. Any specific reason why from should be added here as well?